pylatex/utils.py

```diff
@@ -249,54 +236,304 @@ def bold(s, *, escape=True):
     return NoEscape(r'\textbf{' + s + '}')
 
 
+def text_color(s, color, *, escape=True):
+    r"""Change the string color.
+
+    Args
+    ----
+    s : str
+        The string to be formatted
+    escape : bool
+        If true the text will be escaped
+    color : str
+        The name of the color for the text
+    """
+
+    if escape:
+        s = escape_latex(s)
+
+    return NoEscape(r"\textcolor{" + color + "}{" + s + "}")
+
+
 def italic(s, *, escape=True):
     r"""Make a string appear italicized in LaTeX formatting.
 
     italic() wraps a given string in the LaTeX command \textit{}.
-
     Args
     ----
     s : str
         The string to be formatted.
     escape: bool
         If true the italic text will be escaped
-
     Returns
     -------
     NoEscape
         The formatted string.
-
     Examples
     --------
     >>> italic("hello")
     '\\textit{hello}'
     >>> print(italic("hello"))
     \textit{hello}
-
     """
     if escape:
         s = escape_latex(s)
 
     return NoEscape(r'\textit{' + s + '}')
 
 
+def page_break():
```

Comment:
I like the fact that common commands are added to the library. However, I'd rather have them as subclasses of CommandBase. Please place them in a separate submodule as well. Call it "basic".